### PR TITLE
Implemented `SoundPlayer::InitSoundBuffers`

### PR DESCRIPTION
**Pull request by PieVieRo**
I don't know what I should do with `char *g_SFXList[26]` so i just left it blank

### CODE REVIEW

File: src/i18n.tpl [hunk 1]
```diff
@@ -60,6 +60,7 @@
 #define TH_JP_ERR_MAINMENU_LOAD_SELECT_SCREEN_FAILED "セレクト画面の読み込みに失敗\n"
 #define TH_JP_ERR_SOUNDPLAYER_FAILED_TO_CREATE_BGM_SOUND_BUFFER "error : ストリーミング用サウンドバッファを作成出来ませんでした\n"
 #define TH_JP_ERR_SOUNDPLAYER_FAILED_TO_INITIALIZE_OBJECT "DirectSound オブジェクトの初期化が失敗したよ\n"
+#define TH_JP_ERR_SOUNDPLAYER_FAILED_TO_LOAD_SOUND_FILE "error : Sound ファイルが読み込めない データを確認 %s\n"
```
Comment (roblabla) [inline src/i18n.tpl:63]: Please add the english translation and the generic variant.

File: src/SoundPlayer.cpp [hunk 2]
```diff
@@ -151,4 +154,41 @@ ZunResult SoundPlayer::LoadWav(char *path)
...
+    {
+        for (idx = 0; idx < 3; idx++)
+        {
+            this->soundBuffersToPlay[idx] = -1;
+        }
+        for (idx = 0; idx < ARRAY_SIZE_SIGNED(g_SFXList); idx++)
+        {
+            if (this->LoadSound(idx, g_SFXList[idx]) != ZUN_SUCCESS)
+            {
+                GameErrorContextLog(&g_GameErrorContext, TH_JP_ERR_SOUNDPLAYER_FAILED_TO_LOAD_SOUND_FILE,
```
Comment (roblabla) [inline src/SoundPlayer.cpp:178]:
```suggestion
                GameErrorContextLog(&g_GameErrorContext, TH_ERR_SOUNDPLAYER_FAILED_TO_LOAD_SOUND_FILE_FILE,
```

Comment (roblabla) [reply src/SoundPlayer.cpp:178]: This isn't fixed. Did you forget to push?

Comment (PieVieRo) [reply src/SoundPlayer.cpp]: probably

**Pull request merged**